scripts/generate-certificates.sh

```diff
@@ -1,25 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -e
 
-# Load env file
-set -a
-test -f $(dirname $0)/../containers/.env && source $(dirname $0)/../containers/.env
-set +a
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-cd $(dirname $0)/../certificates
+. $DIR/env.sh
 
-if [ -f $PROJECT_NAME.crt ]; then
-    echo "Certificate already exists."
-else
-    subj="/C=CH/ST=FR/L=Fribourg/O=Liip/OU=Pontsun/CN=$PROJECT_NAME.$PROJECT_EXTENSION"
+mkdir -p $PONTSUN_DIR_ETC/certificates/
 
-    openssl genrsa -out $PROJECT_NAME.rootCA.key 4096
-    openssl req -x509 -new -nodes -key $PROJECT_NAME.rootCA.key -sha256 -days 1024 -out $PROJECT_NAME.rootCA.crt -subj "$subj"
-
-    openssl genrsa -out $PROJECT_NAME.key 4096
-    openssl req -new -sha256 -subj "$subj" -key $PROJECT_NAME.key -out $PROJECT_NAME.csr -config ../config/openssl.cnf
-    openssl x509 -req -in $PROJECT_NAME.csr -CA $PROJECT_NAME.rootCA.crt -CAkey $PROJECT_NAME.rootCA.key -CAcreateserial -out $PROJECT_NAME.crt -days 365 -extensions v3_req -extfile ../config/openssl.cnf
-
-    cat $PROJECT_NAME.crt $PROJECT_NAME.key > $PROJECT_NAME.pem
-    chmod 600 $PROJECT_NAME.key $PROJECT_NAME.pem
-fi
+docker run --rm -v $DIR/../:/generate/ -v $PONTSUN_DIR_ETC/certificates/:/certs/ -it liip/pontsun-helper:latest /generate/scripts/helper/docker-generate-certificates.sh $1
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
If we start to support explicitly Mac, shouldn't we support other platforms also ?

Reply:
Sure. Windows support is needed/wanted, too. But I need the help of our windows users here and something that can be added later.

We supported mac before anyway already (the source of this scripts and the docs/ at least ;))

Reply:
[smallstep CLI](https://smallstep.com/cli/) has a command to install certificates into truststores for windows, linux and macOS. 
Respective code is at [smallstep/truststore](https://github.com/smallstep/truststore):
- [macOS](https://github.com/smallstep/truststore/blob/master/truststore_darwin.go)
- [linux](https://github.com/smallstep/truststore/blob/master/truststore_linux.go)
- [windows](https://github.com/smallstep/truststore/blob/master/truststore_windows.go)

Reply:
It's now included and used, if installed (didn't check on windows, but that's something to be added later)

Reply:
It = smallstep CLI